<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnimationControler</class>
 <widget class="QWidget" name="AnimationControler">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>449</width>
    <height>47</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QToolButton" name="toolButtonPlay">
       <property name="toolTip">
        <string>Play</string>
       </property>
       <property name="text">
        <string>Play</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/images/imageviewer/imageviewer.qrc">
         <normaloff>:/imageviewer/player_play.png</normaloff>:/imageviewer/player_play.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonPause">
       <property name="toolTip">
        <string>Pause</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/images/imageviewer/imageviewer.qrc">
         <normaloff>:/imageviewer/player_pause.png</normaloff>:/imageviewer/player_pause.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButtonStop">
       <property name="toolTip">
        <string>Stop</string>
       </property>
       <property name="text">
        <string>Stop</string>
       </property>
       <property name="icon">
        <iconset resource="../resources/images/imageviewer/imageviewer.qrc">
         <normaloff>:/imageviewer/player_stop.png</normaloff>:/imageviewer/player_stop.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="horizontalSliderCurrentFrame">
       <property name="toolTip">
        <string>Current Frame</string>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
       <property name="tickInterval">
        <number>2</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSpeed">
       <property name="text">
        <string>Speed:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSpeed">
       <property name="toolTip">
        <string>Speed</string>
       </property>
       <property name="suffix">
        <string>%</string>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
       <property name="value">
        <number>100</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources/images/imageviewer/imageviewer.qrc"/>
 </resources>
 <connections/>
</ui>
